### PR TITLE
[tune] Added word-level language modeling RNN example on Wikitext-2.

### PR DESCRIPTION
**Pull request by visatish**
## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://ray.readthedocs.io/en/latest/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.

### CODE REVIEW

**Comment by visatish** [comment]
@richardliaw

File: python/ray/tune/examples/pbt_word_language_model_example.py [hunk 1]
```diff
@@ -0,0 +1,535 @@
...
+        hyperparam_mutations={
+            "lr": lambda: np.random.uniform(15, 25),
+            "dropout": lambda : np.random.uniform(0, 0.4)
+        })
+
+    run(
+        LanguageModel,
+        name="word_language_model_exp",
+        scheduler=pbt,
+        **{
```
Comment (richardliaw) [inline python/ray/tune/examples/pbt_word_language_model_example.py]: can you simplify this? (i.e., don't use the double asterisk)

**Review by richardliaw — CHANGES_REQUESTED** [review]
Can you see https://github.com/ray-project/ray/commit/9f2362041232df20110ed03c0d22ec51ef94edd8 as to how to add an example? 

A couple things to do:

1. add this to the documentation
2. add this to tests

File: python/ray/tune/examples/pbt_word_language_model_example.py [hunk 2]
```diff
@@ -0,0 +1,534 @@
...
+    def __len__(self):
+        return len(self.idx2word)
+
+
+class Corpus(object):
+    def __init__(self, path):
+        self.dictionary = Dictionary()
+        self.train = self.tokenize(os.path.join(path, 'train.txt'))
+        self.valid = self.tokenize(os.path.join(path, 'valid.txt'))
+        self.test = self.tokenize(os.path.join(path, 'test.txt'))
```
Comment (richardliaw) [inline python/ray/tune/examples/pbt_word_language_model_example.py]: Also, can you make sure you do the linting?

https://ray.readthedocs.io/en/latest/tune-contrib.html

**Pull request closed without merging**